### DISCUSSION TITLE
No RSA Certificate file found

### DISCUSSION THREAD

**Discussion by abird** [iammuho/NatterNet#23] [General]
I followed the installation instructions and when I do "go run ." I get a "No RSA Certificate file found" message. I'm running on MacOS 13.4.1.

**Comment by iammuho** [comment]
Hey @abird,

Thank you for this good case! I've converted to the issue and explained how to generate RSA keys and set the paths here:

https://github.com/iammuho/NatterNet/wiki/Getting-Started#setting-up-rsa-keys-for-jwt

I hope that works.

**Reply by abird** [reply]
Thank you!

**Reply by abird** [reply]
I tried your instructions for generating the RSA keys, and I still get the same problem.

I installed the key files in /etc/ssl/certs and when that didn't work, I set the environment variables.

% echo $JWT_PUBLIC_KEY_PATH
/etc/ssl/certs/public.pem
% echo $JWT_PRIVATE_KEY_PATH
/etc/ssl/certs/private.pem
% go run .                  
2023/08/30 16:10:59 No RSA Certificate file found
exit status 1
%

**Reply by iammuho** [reply]
Hey @abird,

It seems package we used "JOSE" for signing, generate etc has been archived and it's only working on me, because I've created before. I've created the issue and will implement a better solution today also will include the solution into an _hack, so setup will be easier. Please follow up the issue.

**Reply by iammuho** [reply]
Hey @abird ,

Change the package completely and included an _hack/rsa.sh file to setup easier. You can re-read the wiki and it should be working. Don't forget to pull the main again.

**Reply by iammuho** [reply]
Related issue: https://github.com/iammuho/NatterNet/issues/25
